src/pages/index.tsx

```diff
@@ -1,81 +1,169 @@
-import { magic } from "lib/magic-client";
-import { useEffect, useState } from "react";
-import { Card, CardSection } from "src/components/index";
-import { Login } from "src/components/index";
-import { Header } from "src/components/index"
-import { Footer } from "src/components/index"
-import { Spinner } from "src/components/index";
-import { getVideos } from "lib/videos";
-import Link from "next/link";
+import { magic } from "lib/magic-client"
+import { useContext, useEffect, useState } from "react"
+import { Card, Login, Header, Footer, Spinner } from "src/components/index"
+import { getVideos } from "lib/videos"
+import Link from "next/link"
+import { CreatorContext } from "src/context"
+import { constants } from "lib/constants"
 
-export default function HomePage() {
+type VideoType = {
+  id: string
+  imgUrl: string
+  title: string
+}
+export default function HomePage(): JSX.Element {
```

Comment:
Wrong type jsx.element please refactor this to 

const function homepage: ( add proper type ) = ()

Export default HomePate